core/modules/field/tests/setgroup_field_tests.lua

```diff
@@ -0,0 +1,102 @@
+local Field = require('field')
```

Comment:
General comment on tests that's not directed at you, but we should probably go through the tests and comment them with their case, their preconditions, and their postconditions.